docs/core/install/linux-package-manager-fedora32.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Install .NET Core on Fedora 32 - package manager - .NET Core
+description: Use a package manager to install .NET Core SDK and runtime on Fedora 32.
+author: thraka
+ms.author: adegeo
+ms.date: 04/28/2020
+---
+
+# Fedora 32 Package Manager - Install .NET Core
+
+[!INCLUDE [package-manager-switcher](./includes/package-manager-switcher.md)]
+
+This article describes how to use a package manager to install .NET Core on Fedora 32.
+
+[!INCLUDE [package-manager-intro-sdk-vs-runtime](includes/package-manager-intro-sdk-vs-runtime.md)]
+
+Starting with Fedora 32, .NET Core 3.1 is available in the default package repositories in Fedora.
+
+## Install the .NET Core SDK
+
+Install the .NET Core SDK. In your terminal, run the following command.
+
+```bash
+sudo dnf install dotnet-sdk-3.1
```

Comment:
Question on this, do you need to ever do `sudo dnf update` ?

Reply:
Only to update the packages you have installed on your disk, I think.

Unlike `apt` where `apt-get update` fetches information about the latest versions of packages so `apt-get install` and `apt-get upgrade` can work with it, `dnf` does it automatically. 

If you do a new Fedora installation (nothing in the package cache) or run a Fedora VM after a few days of not using it, `dnf install dotnet-sdk-3.1` will automatically fetch the information about the latest versions of packages when invoked.